api/src/main/java/org/pmiops/workbench/billing/GoogleApisConfig.java

```
@@ -18,22 +19,42 @@
 @Configuration
 public class GoogleApisConfig {
 
-  @Bean
+  public static final String USER_PROXY_CLOUD_BILLING = "USER_PROXY_CLOUD_BILLING";
+  public static final String SERVICE_ACCOUNT_CLOUD_BILLING = "SERVICE_ACCOUNT_CLOUD_BILLING";
+
+  @Bean(USER_PROXY_CLOUD_BILLING)
   @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
-  public Cloudbilling googleCloudBillingApi(
+  public Cloudbilling userProxyGoogleCloudbillingApi(
       UserAuthentication userAuthentication,
       JsonFactory jsonFactory,
       Provider<WorkbenchConfig> workbenchConfigProvider) {
-    GoogleCredential credential =
-        new GoogleCredential()
-            .setAccessToken(userAuthentication.getCredentials())
-            .createScoped(
-                Collections.singletonList("https://www.googleapis.com/auth/cloud-platform"));
+    return createCloudbillingClient(
+        userAuthentication.getCredentials(), jsonFactory, workbenchConfigProvider.get());
+  }
+
+  @Bean(SERVICE_ACCOUNT_CLOUD_BILLING)
+  @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
+  public Cloudbilling serviceAccountGoogleCloudbillingApi(
+      JsonFactory jsonFactory, Provider<WorkbenchConfig> workbenchConfigProvider) {
+    String accessToken;
+    try {
+      accessToken =
+          ServiceAccounts.getScopedServiceAccessToken(
+              Collections.singletonList("https://www.googleapis.com/auth/cloud-platform"));
+    } catch (IOException e) {
+      throw new RuntimeException("Could not create service account access token for cloud billing");
```

nit: Per other  PR comments, would just let the exception propagate